pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -15,12 +17,55 @@ var _ = Describe("Generating fluentd config", func() {
 		forwarder     *logging.ClusterLogForwarderSpec
 		forwarderSpec *logging.ForwarderSpec
 		generator     *ConfigGenerator
+		secrets       map[string]*corev1.Secret
 	)
 	BeforeEach(func() {
 		var err error
 		generator, err = NewConfigGenerator(false, false, true)
 		Expect(err).To(BeNil())
 		Expect(generator).ToNot(BeNil())
+		secrets = map[string]*corev1.Secret{
+			"infra-es": {
+				ObjectMeta: v1.ObjectMeta{
+					Name: "my-infra-secret",
+				},
+				Data: map[string][]byte{
```

Comment:
nit... one could move this map into a single var for reuse...

Reply:
Fixed